sdk/js/src/rpc/getGovernorIsVAAEnqueuedWithRetry.ts

```diff
@@ -0,0 +1,35 @@
+import { ChainId, ChainName, getGovernorIsVAAEnqueued } from "..";
+import { coalesceChainId } from "../utils";
+
+export async function getGovernorIsVAAEnqueuedWithRetry(
+  hosts: string[],
+  emitterChain: ChainId | ChainName,
+  emitterAddress: string,
+  sequence: string,
+  extraGrpcOpts = {},
+  retryTimeout = 1000,
+  retryAttempts?: number
+) {
+  let currentWormholeRpcHost = -1;
+  const getNextRpcHost = () => ++currentWormholeRpcHost % hosts.length;
+  let result;
```

Comment:
Is this guaranteed to be initialized to a false equivalent or should it be explicitly initialized so that the while loop is guaranteed to run the first time?

Reply:
`result` === `undefined` here so the while loop will exec the first time